### PR TITLE
MMT-1418-Fixed the issue where tophat2 was pushing rest of webpage down and hiding part of the footer

### PR DESCRIPTION
**Pull request by zjam1**
Added a div tag inside of wrapper to push down header. Also changed top margins for the wrapper class.

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -31,5 +31,6 @@
 //= require jquery.webui-popover/jquery.webui-popover.min
 //= require redactor/redactor.min
 //= require jquery-textcounter/textcounter.min
+//= require tophat_space.coffee
```
Comment (charleshuang80) [inline app/assets/javascripts/application.js]: Does this need to be added? I think it should be automatically picked up with the `require_tree` below.

File: app/assets/javascripts/tophat_space.coffee [hunk 2]
```diff
@@ -0,0 +1,5 @@
+$('document').ready ->
+  #checks if tophat exists 
+  tophatExist = document.getElementsByClassName("th-wrapper")
```
Comment (charleshuang80) [inline app/assets/javascripts/tophat_space.coffee]: Sorry I didn't notice this before, but you should make this call with JQuery, which you are using above and below

Comment (jceaser) [reply app/assets/javascripts/tophat_space.coffee]: yea, that is a good point. https://stackoverflow.com/questions/23648196/converting-document-getelementsbyclassname-to-jquery has an example of this.

**Comment by zjam1** [comment]
I made the changes that Charles requested

File: app/assets/javascripts/tophat_space.coffee [hunk 3]
```diff
@@ -0,0 +1,5 @@
+$('document').ready ->
+  # checks if tophat exists
+  tophatExist = $(".th-wrapper")
+  if tophatExist.length
```
Comment (charleshuang80) [inline app/assets/javascripts/tophat_space.coffee]: I was going to say you should add a ` > 0` here, but upon further research it looks like this is a widely accepted way to do this. However, I will suggest one final change, you can shorten this by making it one line: `if $('.th-wrapper).length`

**Pull request closed without merging**